hatchet/graphframe.py

```diff
@@ -32,6 +32,16 @@
     raise
 
 
+def graphframe_reader(func):
```

Comment:
I think we should rename this decorator to better reflect its purpose, I think the inner function is on the right track. Maybe `@store_graphframe_input_dataset`?

Reply:
Fixed by bd7d4c9